docs/filemanagement.rst

```diff
@@ -172,6 +172,25 @@ one permission at a time by providing the permission's ID.
 
     file1.DeletePermission(permission_id)  # Delete the permission.
 
+Get files by complex queries
+----------------------------
+
+We can get a file by name and by other constraints, usually a filename will be unique but
+we can have two equal names with different extensions, e.g.,  *123.jpeg and 123.mp3*. So if you
+expect only one file add more constraints to the query, see `q_parameters`_,
+as a result we get a list of `GoogleDriveFile`_ instances.
+
+.. code-block:: python
+
+    from pydrive2.drive import GoogleDrive
+    # Create GoogleDrive instance with authenticated GoogleAuth instance.
+    drive = GoogleDrive(gauth)
+    filename = 'file_test'
+    # query
```

Comment:
please try to keep the style the same (as far as I can see).

E.g. comments start with a capital letter:

`# The permissions array is now available for further use.`